assets/js/components/notifications/ZeroDataStateNotifications.js

```diff
@@ -99,26 +108,28 @@ export default function ZeroDataStateNotifications() {
 				/>
 			) }
 
-			{ ( analyticsHasZeroData || searchConsoleHasZeroData ) && (
-				<BannerNotification
-					id="zero-data-notification"
-					title={ __(
-						'Not enough traffic yet to display stats',
-						'google-site-kit'
-					) }
-					description={ __(
-						'Site Kit will start showing stats on the dashboard as soon as enough people have visited your site. Keep working on your site to attract more visitors.',
-						'google-site-kit'
-					) }
-					format="small"
-					learnMoreLabel={ __( 'Learn more', 'google-site-kit' ) }
-					learnMoreURL="https://sitekit.withgoogle.com/documentation/using-site-kit/dashboard-data-display/"
-					dismiss={ __( 'Remind me later', 'google-site-kit' ) }
-					isDismissible
-					dismissExpires={ getTimeInSeconds( 'day' ) }
-					SmallImageSVG={ ZeroStateIcon }
-				/>
-			) }
+			{ ( analyticsGatheringData === false ||
+				searchConsoleGatheringData === false ) &&
```

Comment:
This should be an `AND` conditional, shouldn't it? Because otherwise if only one of these is `false` and the other one `true`, eg. `analyticsGatheringData === false && searchConsoleGatheringData === true`, `searchConsoleHasZeroData` would also be `true` and React would render this.

As mentioned in the IB: our UI won't actually be updated because we only show one notification at a time, but in this case the bug this attempts to solve could still happen if we weren't returning early like above.

This shouldn't ever happen, but these conditions _imply_ it'd be okay if they did… or even that maybe we think they _should_. So this should probably be:

```suggestion
			{ ( analyticsGatheringData === false &&
				searchConsoleGatheringData === false ) &&
```

Reply:
>This should be an AND conditional, shouldn't it?

Thanks, @tofumatt. You are right 💯  

Let me update the condition, and let's have it since it explicitly states when to render the zero data notifications 👍